src/Articulate.Web/config/grid.editors.config.js

```diff
@@ -1,4 +1,4 @@
-﻿[
+[
```

Comment:
Also all this file is part of the umbraco starter pack... should have not been committed